plugins/dynatrace/.eslintrc.js

```diff
@@ -1 +1,5 @@
-module.exports = require('@backstage/cli/config/eslint-factory')(__dirname);
+module.exports = require('@backstage/cli/config/eslint-factory')(__dirname, {
+    rules: {
+      '@backstage/no-top-level-material-ui-4-imports': 'error',
+    },
+  });
```

Comment:
OK sorry but there's going to be SO MANY of these. Can you please enable "format on save" or similar in your editor? This is messing up the formatting of every single eslintrc file in the repo :)

See also the [comment in the other PR](https://github.com/backstage/backstage/pull/23530#discussion_r1524644688) about shortening the changesets